src/mainwindow.cpp

```diff
@@ -1932,11 +1932,21 @@ void MainWindow::refreshSaveList()
 
   QDir savesDir = currentSavesDir();
   savesDir.setNameFilters(filters);
+  savesDir.setFilter(QDir::Files);
+  QDirIterator it(savesDir, QDirIterator::Subdirectories);
   log::debug("reading save games from {}", savesDir.absolutePath());
 
-  QFileInfoList files = savesDir.entryInfoList(QDir::Files, QDir::Time);
+  QFileInfoList files;
+  while (it.hasNext()) {
+    it.next();
+    files.append(it.fileInfo());
+  }
+  std::sort(files.begin(), files.end(), [](auto const& lhs, auto const& rhs) { 
+    return lhs.fileTime(QFileDevice::FileModificationTime) < rhs.fileTime(QFileDevice::FileModificationTime); 
```

Comment:
Did we look at the file time even before? Could this slow down refresh times with big saves folders?

Reply:
Yes, it was in the `savesDir.entryInfoList(QDir::Files, QDir::Time)` (the `QDir::Time`) but you cannot do that directly with `QDirIterator`.

Reply:
Ok, thanks. Not for this PR, do you think it could be possible to perform the saves refresh on a different thread asynchronously?

Reply:
Probably, it's just reading the files and populating the list.